examples/lambda/broker.py

```diff
@@ -0,0 +1,21 @@
+from lbz.lambdas import LambdaBroker, LambdaResponse, lambda_ok_response
+from lbz.types import LambdaContext
+
+
+def get_user_data(data: dict) -> LambdaResponse:
+    print(data)
+    return lambda_ok_response()
```

Comment:
It would be good to return some data here, especially since you rely on that in the `examples/lambda/client.py` example 😉